examples/src/main/scala/org/apache/spark/examples/HdfsTest.scala

```diff
@@ -41,6 +41,8 @@ object HdfsTest {
       val end = System.currentTimeMillis()
       println(s"Iteration $iter took ${end-start} ms")
     }
+    println(s"File contents: ${file.map(s => s.toString).collect().mkString(",")}")
```

Comment:
Spark's convention is either `.map { s=> s.toString }` or `.map(_.toString)`. I noticed this in a few other places too.